app/modules/intelligence/agents/chat_agents/tool_helpers.py

```
@@ -0,0 +1,178 @@
+        case "GetNodesfromTags":
+            return ""
+        case "AskKnowledgeGraphQueries":
+            queries = args.get("queries")
+            if isinstance(queries, List):
+                return "".join([f"\n- {query}" for query in queries])
+            return ""
+        case "GetCodeanddocstringFromMultipleNodeIDs":
+            return ""
+        case "get_code_file_structure":
+            path = args.get("path")
+            if path != None and isinstance(path, str):
```

Ruff (0.8.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)